packages/web/src/components/upload/TrackPreview.module.css

```diff
@@ -97,11 +97,16 @@
 }
 
 .trackPreviewNew:hover .fileSizeText {
-  opacity: 0;
+  display: none;
 }
 
-.trackPreviewNew:hover .removeButton {
-  opacity: 1;
+.trackPreviewNew:hover .iconsContainer {
+  width: 80px;
```

Comment:
nit: a bit code smell but if it's needed then it's needed

Reply:
yeah it's to make sure the container doesnt move. will think of another solution, if nothing then will keep this